vhdbuilder/packer/test/vhd-content-test.ps1

```diff
@@ -149,7 +149,7 @@ function Test-FilesToCacheOnVHD
 function Test-PatchInstalled
 {
```

Comment:
Could you help to update the base image to `17763.1637.2012040632`?
WindowsServer  MicrosoftWindowsServer  2019-Datacenter-Core-smalldisk     MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-smalldisk:17763.1637.2012040632     17763.1637.2012040632

Reply:
It's not a must. We can update the image to stable 12B after it's released

Reply:
1204 was released before 12B. I think that we should always follow the principal to install the latest security patch in the latest Windows OS image.

Reply:
Do we have such a principle? Take a look at this please https://github.com/Azure/aks-engine/pull/3692

Reply:
It is always better to check the base image and then update the image before installing a new patch. You can go ahead if you fully understand what it is. Otherwise, we need more effort to clarify why we do not install the latest patch in the latest base image but in an older base image.

Reply:
If you think we may lost something, I don't mind we have more test in vhdtest to check.